cargo audit fails

Failure on `dev` as follows:
```
Crate:         crossbeam-deque
Version:       0.7.3
Title:         Data race in crossbeam-deque
Date:          2021-07-30
ID:            RUSTSEC-2021-0093
URL:           https://rustsec.org/advisories/RUSTSEC-2021-0093
Solution:      Upgrade to >=0.7.4, <0.8.0 OR >=0.8.1
Dependency tree: 
crossbeam-deque 0.7.3

Crate:         crossbeam-deque
Version:       0.8.0
Title:         Data race in crossbeam-deque
Date:          2021-07-30
ID:            RUSTSEC-2021-0093
URL:           https://rustsec.org/advisories/RUSTSEC-2021-0093
Solution:      Upgrade to >=0.7.4, <0.8.0 OR >=0.8.1
Dependency tree: 
crossbeam-deque 0.8.0

Crate:         blake2b
Version:       0.1.0
Warning:       yanked
Dependency tree: 
blake2b 0.1.0

Crate:         crossbeam-deque
Version:       0.7.3
Warning:       yanked

Crate:         crossbeam-deque
Version:       0.8.0
Warning:       yanked

error: 2 vulnerabilities found!
warning: 3 allowed warnings found
make: *** [Makefile:120: audit] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -117,7 +117,7 @@ lint: lint-contracts-rs
 
 .PHONY: audit
 audit:
-	$(CARGO) audit --ignore RUSTSEC-2021-0073 --ignore RUSTSEC-2021-0076
+	$(CARGO) audit
 
 .PHONY: doc
 doc:

--- a/execution_engine/Cargo.toml
+++ b/execution_engine/Cargo.toml
@@ -52,7 +52,7 @@ wasmi = "0.8.0"
 
 [dev-dependencies]
 assert_matches = "1.3.0"
-criterion = "0.3.3"
+criterion = "0.3.5"
 proptest = "1.0.0"
 tempfile = "3.1.0"
 

--- a/execution_engine_testing/tests/Cargo.toml
+++ b/execution_engine_testing/tests/Cargo.toml
@@ -27,7 +27,7 @@ tempfile = "3"
 
 [dev-dependencies]
 assert_matches = "1.3.0"
-criterion = "0.3.0"
+criterion = "0.3.5"
 gh-1470-regression = { path = "../../smart_contracts/contracts/test/gh-1470-regression", default-features = false, features = ["std"] }
 gh-1470-regression-call = { path = "../../smart_contracts/contracts/test/gh-1470-regression-call", default-features = false, features = ["std"] }
 num-rational = "0.4.0"

--- a/smart_contracts/contracts/test/blake2b/Cargo.toml
+++ b/smart_contracts/contracts/test/blake2b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "blake2b"
-version = "0.1.0"
+version = "0.8.0"
 authors = ["Henry Till <henrytill@gmail.com>"]
 edition = "2018"
 

--- a/types/Cargo.toml
+++ b/types/Cargo.toml
@@ -37,7 +37,7 @@ uint = { version = "0.9.0", default-features = false }
 
 [dev-dependencies]
 bincode = "1.3.1"
-criterion = "0.3.3"
+criterion = "0.3.5"
 getrandom = { version = "0.2.0", features = ["rdrand"] }
 proptest = "1.0.0"
 serde_json = "1.0.55"

